chain/transaction_test.go

```diff
@@ -0,0 +1,155 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package chain_test
+
+import (
+	"context"
+	"encoding/hex"
+	"testing"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/hypersdk/auth"
+	"github.com/ava-labs/hypersdk/chain"
+	"github.com/ava-labs/hypersdk/codec"
+	"github.com/ava-labs/hypersdk/crypto/ed25519"
+	"github.com/ava-labs/hypersdk/state"
+)
+
+var (
+	_ chain.Action = (*mockTransferAction)(nil)
+	_ chain.Action = (*action2)(nil)
+)
+
+type mockTransferAction struct {
+	To    codec.Address `serialize:"true" json:"to"`
+	Value uint64        `serialize:"true" json:"value"`
+	Memo  []byte        `serialize:"true" json:"memo"`
+}
+
+type action2 struct {
+	A uint64 `serialize:"true" json:"a"`
+	B uint64 `serialize:"true" json:"b"`
+}
+
+func (*action2) ComputeUnits(chain.Rules) uint64 {
+	panic("unimplemented")
+}
+
+func (*action2) Execute(_ context.Context, _ chain.Rules, _ state.Mutable, _ int64, _ codec.Address, _ ids.ID) (outputs [][]byte, err error) {
+	panic("unimplemented")
+}
+
+func (*action2) GetTypeID() uint8 {
+	return 222
+}
+
+func (*action2) StateKeys(_ codec.Address, _ ids.ID) state.Keys {
+	panic("unimplemented")
+}
+
+func (*action2) StateKeysMaxChunks() []uint16 {
+	panic("unimplemented")
+}
+
+func (*action2) ValidRange(chain.Rules) (start int64, end int64) {
+	panic("unimplemented")
+}
+
+func (*mockTransferAction) ComputeUnits(chain.Rules) uint64 {
+	panic("ComputeUnits unimplemented")
```

Comment:
```suggestion
	panic("unimplemented")
```
saying "function-name unimplemented" is redundant as in the error logs, it'll show the function as the one responsible for panicking.